liquibase-couchbase/src/test/java/liquibase/ext/couchbase/change/MutateInChangeTest.java

```diff
@@ -46,9 +47,10 @@ void Should_parse_mutateIn_id_correctly() {
         assertThat(changeSet.getChanges())
                 .map(MutateInChange.class::cast)
                 .containsExactly(
-                        changeWithId(asList(spec("user.age", "29", DataType.STRING, MutateInType.INSERT))),
-                        changeWithWhereClause(asList(spec("adoc", "{\"newDocumentField\": \"newDocumentValue\"}", DataType.JSON, MutateInType.REPLACE))),
-                        changeWithSqlPlusPlusQuery(asList(spec("user.age", "50", DataType.STRING, MutateInType.INSERT)))
+                        changeWithId(Collections.singletonList(spec("user.age", "29", DataType.STRING, MutateInType.INSERT))),
```

Comment:
Pls, use static import for `singletonList()`

Reply:
done